### DISCUSSION TITLE
Pluto keeps running into "UndefVarError: .... not defined" after some times

### DISCUSSION THREAD

**Discussion by dmetivie** [fonsp/Pluto.jl#2535] [Questions about using Pluto]
Pluto works fine and randomly after some time and usage (like every 10 mins sometimes), it suddenly declares
```
UndefVarError: a_variable_name not defined
```
even if the cell where `a_variable_name` is defined has not been touched.
ReRunning the cell `a_variable_name` most often shift the problem to another variable, e.g.  
```
UndefVarError: a_previous_variable_name not defined
```
doing that iteratively, I typically end up rerunning 
`using SomePackage` at the top of the notebook.

I don't understand where this comes from, and I don't have a MWE since this seems to be quite random (I'll try to post a video next time it happens).

Anyone else encounters that type of issue?

**Comment by Pangoraw** [comment]
Hi, your problem looks definitely related to something like https://github.com/fonsp/Pluto.jl/issues/2090 which is a known bug. It is hard to determine the precise causes without a MWE or a runnable notebook.
